README.md

```
@@ -227,44 +227,41 @@ make example
-     where $\epsilon" is a small perturbation.
+
+     $$\nabla_\theta \mathbb{E}[f(x; \theta)] \approx \frac{\mathbb{E}[f(\theta + \epsilon)] - \mathbb{E}[f(\theta - \epsilon)]}{2\epsilon}$$
+
+     where $\epsilon$ is a small perturbation.
    - **REINFORCE**:
-     $$
-     \nabla_\theta \mathbb{E}[f(x; \theta)] \approx \mathbb{E}\left[f(x) \cdot \nabla_\theta \log \pi(x | \theta)\right].
-     $$
+
+     $$\nabla_\theta \mathbb{E}[f(x; \theta)] \approx \mathbb{E}\left[f(x) \cdot \nabla_\theta \log \pi(x | \theta)\right]$$
+     ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)